components/modal/modal.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modal | BoostMat UI </title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="modal.css">
+    <link rel="stylesheet" href="../component.css">
+    <link rel="stylesheet" href="../main.css">
```

Comment:
Just a suggestion. You can import all the css into one css file and link that one file in html.

Reply:
Thank You, will adjust it